docs/tutorials/policies.md

```diff
@@ -0,0 +1,218 @@
+## Jiva Volume Policies:
+
+Jiva Volumes can be provisioned based on different policy configurations. JivaVolumePolicy has to be created prior to StorageClass 
+and we have to mention the `JivaVolumePolicy` name in StorageClass parameters to provision jiva volume based on configured policy.
+
+Following are list of policies that can be configured based on the requirements.
+
+- [replicationFactor](#replication-factor)
+- [Replica Affinity](#replica-affinity)
+- [Target pod Affinity](#target-pod-affinity)
+- [Resource Request and Limits](#resource-request-and-limits)
+- [Priority Class](#priority-class)
+
+Below StorageClass example contains `jivaVolumePolicy` parameter having `example-jivavolumepolicy` name set to configure the custom policy.
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: openebs-jiva-csi-sc
+provisioner: jiva.csi.openebs.io
+allowVolumeExpansion: true
+parameters:
+  cas-type: "jiva"
+  jivaVolumePolicy: "example-jivavolumepolicy"
+```
+
+### Replication Factor:
+
+Replication factor can be set based on the number of copies of the data required to be maintained.
+If not provided, the replicationFactor is set to 3 by default.
+
+```yaml
+apiVersion: openebs.io/v1alpha1
+kind: JivaVolumePolicy
+metadata:
+  name: example-jivavolumepolicy
+  namespace: openebs
+spec:
+  target:
+    replicationFactor: 3
+```
+
+### Replica Affinity:
```

Comment:
we don't have replica affinity in case of jiva @payes , we can remove this section